components/SensorComponent.js

```diff
@@ -0,0 +1,115 @@
+import React, { useState, useEffect } from "react";
+import { View, Image, StyleSheet } from 'react-native';
+import { Accelerometer } from "expo-sensors";
+import styled from "styled-components/native";
+
+const styles = StyleSheet.create({
+
+  Image: {
+    width: 350,
+    height: 350,
+    borderRadius: 10,
+  },
+});
```

Comment:
Interesting to see how you added the styling of the Image in an own component. I styled it directly in the Image Style (line 73 below in your component). But was wondering a bit if I wanted to do more styling - how would adress it. Thanks!